src/Common/tests/System/Collections/ISet.Generic.Tests.cs

```diff
@@ -419,6 +419,7 @@ public void ISet_Generic_ExceptWith_Itself(int setLength)
 
         [Theory]
         [MemberData(nameof(ValidCollectionSizes))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
Can we add comment? Either with link to a bug which tracks the port, or comment stating it won't ever happen (potentially even with link to a bug where it is discussed).